indico/modules/events/papers/templates/_contributions.html

```diff
@@ -31,6 +36,56 @@ <h3 class="icon-file">{% trans %}Paper{% endtrans %}</h3>
     {% endif %}
 {% endmacro %}
 
+{% macro render_editables_section(contribution) %}
+    {% set editable = contribution.editables[0] if contribution.editables if contribution.editables else None %}
+    {% set show_editable = true if editable else false %}
+    {% set can_submit = contribution.is_user_associated(session.user, check_abstract=true) %}
+
+    {% if can_submit %}
+        <section>
+            <div class="header">
+                <div class="header-row">
+                <h2>{% trans %}Editing{% endtrans %}</h2>
+                </div>
+            </div>
+            <div class="header">
+                <div class="header-row">
+                    <h3 class="icon-file">{% trans %}Paper{% endtrans %}</h3>
+                </div>
+            </div>
+            {% if show_editable %}
+                <div>
+                    <a href="{{ url_for('event_editing.editable', contribution.event, contrib_id=contribution.id, type='paper') }}">
+                        {%- trans %}Go to timeline{% endtrans -%}
+                    </a>
+                </div>
+            {% else %}
+                <div class="action-box highlight">
+                    <div class="section flexrow">
+                        <div class="icon icon-file"></div>
+                            <div class="text">
+                                <div class="label">
+                                    {% trans %}Paper submission is open{% endtrans %}
+                                </div>
+                                <div>
+                                    {% trans %}You can submit a paper{% endtrans %}
+                                </div>
+                            </div>
+                        <div class="toolbar">
+                            <div id="editable-submission-button"
+                                data-event-id="{{ contribution.event_id }}"
+                                data-contribution-id="{{ contribution.id }}"
+                                data-type="paper"
+                                data-upload-url="{{ url_for('event_editing.api_upload', contribution.event, contrib_id=contribution.id, type='paper') }}"
```

Comment:
`contribution.event, contrib_id=contribution.id` can be replaced with `contribution` (its locator provides both values)